doc/faq.rst

```diff
@@ -0,0 +1,40 @@
+Frequently asked questions
+==========================
+
+Should I apply ICA first or autoreject first?
+---------------------------------------------
+
+ICA solutions can be affected by high amplitude artifacts, therefore
+we recommend first using autoreject to detect the bad segments, then applying
+ICA, and finally interpolating the bad data::
+
+	>>> ar = Autoreject()
+	>>> ar.fit(epochs)
+	>>> ica.fit(epochs[~ar.reject_log.bad_epochs_idx])
+	>>> ica.exclude = [5, 7]  # exclude EOG components
+	>>> ica.transform(epochs)
+	>>> ar.transform(epochs)
+
+Autoreject is not meant for eyeblink artifacts since it affects neighboring
+sensors. Indeed, a spatial filtering method like ICA is better suited for this.
+
```

Comment:
What I have been doing in fact quite often was to use the global thresholds, pass them as reject params to ICA and then estimate ICA on raw. Then use autoreject for epochs and apply ICA whenever desired.
Maybe we should also point out that option because it now sounds as if this was the only way to do it.